addons/hr/static/src/scss/hr.scss

```diff
@@ -77,3 +77,7 @@
         max-width: #{$size}rem!important;
     }
 }
+
+th[data-name="date_version"] {
```

Comment:
This rule is gonna be evaluated everywhere, no only in the HR context.

This could increase the page loading time, and also apply this style where the ``th[data-name="date_version"]`` selector is matched outside of this specific context.

Couldn't you complexify the selector a little bit, like
```css
.o_hr_employee_list_view th[data-name="date_version"]
``` 
Note: This is untested, I don't even know if the ``o_hr_employee_list_view`` class exists.